cucumber-core/src/main/java/io/cucumber/core/order/factory/LexicalUriPickleOrderFactory.java

```diff
@@ -0,0 +1,17 @@
+package io.cucumber.core.order.factory;
+
+import io.cucumber.core.order.PickleOrder;
+import io.cucumber.core.order.StandardPickleOrders;
+
+public class LexicalUriPickleOrderFactory implements PickleOrderFactory {
+
+    @Override
+    public String getName() {
+        return "normal";
+    }
+
+    @Override
+    public PickleOrder create(String argument) {
+        return StandardPickleOrders.lexicalUriOrder();
```

Comment:
Feel free to inline the implementation from `StandardPickleOrders` and have `StandardPickleOrders` invoke the factories. Then we can deprecate `StandardPickleOrders` and remove it in the future.